build.gradle

```diff
@@ -21,6 +22,9 @@ allprojects {
             options.compilerArgs << "-Xlint:unchecked"
         }
     }
+    tasks.withType(Javadoc) {
+        options.addStringOption('Xdoclint:none', '-quiet')
```

Comment:
Java 8's stricter javadoc linting was throwing errors where there used to be warnings, which led to the build failing.